sonic-net/sonic-buildimage
# [docker-sonic-vs]: More changes to support DPU-2P HWKSU

#### Why I did it
- port_config.ini and hwsku.json are needed to generate the default config
- switch_type needs to be "dpu" to spawn the right set of processes during dvs initialization and to make sure that DASH APIs can be handled properly
##### Work item tracking
- Microsoft ADO **24375371**:

#### How I did it
- Use the same hwsku.json and port_config.ini for DPU-2P as the ones used for Nvidia-MBF2H536C SKU in nvidia-sonic sonic-buildimage repo.
- Set switch_type to "dpu" in DEVICE_METADATA configuration to make sure DASH specific APIs are handled properly
#### How to verify it
Spawn a vs testbed using DPU-2P hwksu and make sure DASH vstests are able to run.


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
